js/models/BaseModel.js

```diff
@@ -116,8 +122,16 @@ export default class extends Model {
         const nestedInstance = this.attributes[nestedKey];
 
         if (nestedInstance) {
-          if (nestedData) nestedInstance.set(nestedData);
-          delete attrs[nestedKey];
+          try {
```

Comment:
Improperly set how? AFAIK nested is just a getter returning an object that maps nested key names to class constructors, which is implemented.

I see I used `nested` as a method which is how it is in `Profile.js`. But actually in `listings/*.js` models `nested` is a getter. I think I copied the usage which is a typo and then monkey patched all around that incorrect usage to try to get it to work. I'm going to try and test changing both `Profile.js` and advanced settings usage to the getter usage of nested. I'm very hopeful this might be the cause of the weird bug I saw.